plugins/org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/StandardLanguage.xtend

```diff
@@ -95,6 +88,14 @@ import org.eclipse.xtext.xtext.generator.xbase.XtypeGeneratorFragment2
 	 
 	XtextAntlrIDEAGeneratorFragment ideaParser = new XtextAntlrIDEAGeneratorFragment
 	
+	TypesGeneratorFragment2 commonTypesSupport = new TypesGeneratorFragment2 => [
+		onlyEnabledIfGrammarIsUsed = true
+	]
+	
+	XtypeGeneratorFragment2 xtypeSupport = new XtypeGeneratorFragment2
+	
+	XbaseGeneratorFragment2 xbaseSupport = new XbaseGeneratorFragment2
```

Comment:
Reordered fragment will change bindings in generated code, e..g for rename refactoring if xbase is used.